server/src/parser/slangHelpers.ts

```diff
@@ -0,0 +1,45 @@
+import { NodeType, RuleNode, TokenNode } from "@nomicfoundation/slang/cst";
+import { RuleKind, TokenKind } from "@nomicfoundation/slang/kinds";
+import { TextRange } from "@nomicfoundation/slang/text_index";
+import _ from "lodash";
+import { TextDocument } from "vscode-languageserver-textdocument";
+import { Range } from "vscode-languageserver-types";
+import { getPlatform } from "../utils/operatingSystem";
+
+export type SlangNode = RuleNode | TokenNode;
+export type NodeKind = RuleKind | TokenKind;
+export type NodeCallback = (node: SlangNodeWrapper) => void;
```

Comment:
looks like `NodeCallback` is not used anywhere. Is this intended?

Reply:
Good catch! it was left over from before the rework